11LessonEleven/in-class-project/css/style.css

```diff
@@ -0,0 +1,74 @@
+body {
+    width: 100%;
+    margin: auto;
+}
+#main {
+    text-align: center;
+    background-color: aqua;
+    width: 100%;
+    height:250px;
+    color: white;
+    padding-top: 100px;
+}
+#main h1 {
+    font-size: 3em;
+}
+.clearfix:after {
+    content: "";
+    display: block;
+    clear: both;
+}
+#boxes {
+    width: 100%;
+    height: 325px;
+    background-color: black;
+    display: flex;
+    justify-content: center;
+    padding: 15px 0;
+}
+.left {
+    background-color: black;
+}
+.middle {
+    background-color: black;
+    margin: 0 30px;
+}
+.right {
+    background-color: black;
+}
+.content-box {
+    text-align: center;
+    height: 100%;
+    width: 200px;
+    float: left;
+    color: white;
+    padding: 0 30px;
+}
+.colorboxes {
+    float: left;
+    color: white;
+}
+.colorbar {
+    width: 100%;
+    height: 20px;
+    background-color: white;
+    margin: 50px auto auto auto;
+}
+#left-cbar {
+    background-color: blue;
+}
+#middle-cbar {
+    background-color: red;
+}
+#right-cbar {
+    background-color: aqua;
+}
+
+}
+@media (max-width: 768px) {
```

Comment:
your `@media` it's not working. check where to out the missing closing bracket `}`